feature-group-definitions/js-modules-for-service-workers.yml

```diff
@@ -0,0 +1,7 @@
+name: Service worker JavaScript modules
+description: 'The `navigator.serviceWorker.register()` method accepts `{ type: "module" }` to load scripts that use `import` and `export`. Also known as ECMAScript modules or ESM in service workers.'
+spec:
+  - https://html.spec.whatwg.org/multipage/workers.html#dom-worker-dev
+  - https://html.spec.whatwg.org/multipage/workers.html#shared-workers-and-the-sharedworker-interface:dom-sharedworker-2
+compat_features:
+  - api.ServiceWorker.ecmascript_modules
```

Comment:
Yep, that's where it is. I think the more correct place would be under `api.ServiceWorkerContainer.register`, but arguably that's not where people are going to find it. So let's leave this alone until it changes in BCD, which I'm not going to push for.